control/cli.py

```diff
@@ -108,13 +108,13 @@ def __init__(self):
             required=False)
         self.parser.add_argument(
             "--server-address",
-            default="localhost",
+            default=(os.getenv('CEPH_NVMEOF_SERVER_ADDRESS') or "localhost"),
             type=str,
             help="Server address",
         )
         self.parser.add_argument(
             "--server-port",
-            default=5500,
+            default=(os.getenv('CEPH_NVMEOF_SERVER_PORT') or 5500),
```

Comment:
We should briefly document this (in the CLI usage message) or it'll be just a hidden feature :P